extensions/skins/conversation_v1/player.html

```diff
@@ -126,6 +126,15 @@
                     (<[activeCard.interactionInstructions]>)
```

Comment:
I would actually get rid of the parentheses here, I think it's cleaner without them.